package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "s2-sos",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^12.20.47",
+    "@types/react": "^16.14.24",
+    "@types/react-dom": "^16.9.14",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-responsive-carousel": "^3.2.23",
+    "react-router-dom": "^6.2.2",
+    "react-scripts": "5.0.0",
+    "typescript": "^4.6.2",
+    "web-vitals": "^0.2.4",
```

Comment:
This is just nitpicking, normally the testing stuff, `@types/*` and `typescrip` go into dev dependencies.